src/libpmemfile/preload.c

```diff
@@ -204,6 +204,8 @@ static void establish_mount_points(const char *);
 static void setup_strict_symlink_flag(const char *);
 static void init_hooking(void);
 
+int pause_at_start;
```

Comment:
Might wanna make this `volatile`, just to be sure that loop won't be eliminated.